cyclonedx/model/component.py

```diff
@@ -191,8 +210,9 @@ def licenses(self) -> "SortedSet[LicenseChoice]":
     def licenses(self, licenses: Iterable[LicenseChoice]) -> None:
         self._licenses = SortedSet(licenses)
 
-    @property
-    def copyright(self) -> "SortedSet[Copyright]":
+    @property  # type: ignore[misc]
+    @serializable.xml_array(serializable.XmlArraySerializationType.NESTED, 'text')
+    def copyright_(self) -> "SortedSet[Copyright]":
```

Comment:
:memo: breaking change

:question: why is this renaming done? is there any benefit?

Reply:
To avoid Pythonic keywords therefore ensuring collisions don't occur (part of automagical (de-)serialization.

Reply:
i would say this does not apply on a data model class.
let's overrule this, if it was a rule.